rush/rush-lib/src/utilities/Npm.ts

```diff
@@ -0,0 +1,31 @@
+import Utilities from './Utilities';
+import * as semver from 'semver';
+
+export default class Npm {
+  public static publishedVersions(
+    packageName: string,
+    cwd: string,
+    env: { [key: string]: string }
+  ): string[] {
+    const versions: string[] = [];
+    try {
+      const packageTime: string = Utilities.executeCommandAndCaptureOutput('npm',
+        `view ${packageName} time --json`.split(' '),
```

Comment:
I am considering the `package-json` package to collect this information for evergreen. Perhaps we should do the same thing here?